ticdc/ticdc-alert-rules.md

```diff
@@ -181,7 +181,7 @@ summary: 了解 TiCDC 集群监控报警规则以及处理方法。
 
 * 处理方法：
 
-    MySQL 报错的情况较多，参考[`TiCDC 故障处理`](/ticdc/troubleshoot-ticdc.md)。
+    MySQL 报错的情况较多，参考 [TiCDC 故障处理](/ticdc/troubleshoot-ticdc.md)。
```

Comment:
Similar places in this doc include chapter names by ``, for example, line26, line 40, line 54, line 114. Should I change them as well?
![image](https://user-images.githubusercontent.com/89768198/175857110-e6fe3588-2d65-47c5-99ed-34745545c509.png)